From 25f88970df709c9c641c8c2b7b5a96b334ca7a9c Mon Sep 17 00:00:00 2001
From: Dima Kogan <dima@secretsauce.net>
Date: Mon, 1 Sep 2014 13:03:42 -0700
Subject: [PATCH] <package>.displace-extension can now contain ".extension" or
 "extension"

Previously the former was assumed, and the latter would silently fail
---
 dh_configpackage | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dh_configpackage b/dh_configpackage
index 942766f..0e47367 100755
--- a/dh_configpackage
+++ b/dh_configpackage
@@ -223,8 +223,10 @@ sub displace_extension {
     if ($file) {
         open(my $fh, $file);
         my $ret = <$fh>;
+        close $fh;
+
         chomp $ret;
-        close $fh;
+        $ret = ".$ret" unless $ret =~ /^\./; # must start with .
         return $ret;
     }
     $package =~ s/-.*//;
-- 
2.0.0

